### PR TITLE
add tenant token demo

### LINKED ISSUE #22: Add tenant token demo
As Meilisearch v0.26.0 got released with the feature of tenant tokens, we have created a demo to showcase how to generate and use tenant tokens inside of Meilisearch.

### PR DESCRIPTION
**Pull request by shivaylamba**
# Pull Request

## What does this PR do?
Fixes #22 

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: src/tenant-token/frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
```
Comment (bidoubiwa) [inline src/tenant-token/frontend/README.md:11]:
```suggestion
Runs the app in the development mode.
```

There are a lot of backslashes, are they useful ?

File: src/tenant-token/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+
+
+## 📖 Documentation
+
+See our [Documentation](https://docs.meilisearch.com/learn/security/tenant_tokens.html#what-is-multitenancy) or our [API References](https://docs.meilisearch.com/reference/api/).
+
+## 📝 Requirements
+
+ - [NodeJs](https://nodejs.org/en/download/)
```
Comment (bidoubiwa) [inline src/tenant-token/README.md:39]: You need to add `npm`

File: src/tenant-token/README.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+In this repository, you will find three sub directories:
+  1. **Frontend** - React App to search,create token and switch between tokens.
+  2. **Backend** - Express App to handle Apis.
+  3. **Seed** - Script to feed intial data in meilisearch.
+
+## 🎬 Getting Started
+
+>Prerequisite: Start Docker before going further
+
+We will set up things in sequence to avoid dependencies.
```
[the remark this answers is not in the document]
Comment (shivaylamba) [reply src/tenant-token/README.md:53]: That we need to follow the steps in sequence. I have removed the line to avoid confusion.

**Comment by shivaylamba** [comment]
@bidoubiwa I have made the suggested changes.

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

File: src/tenant-token/README.md (hunk 2 above)
Comment (shivaylamba) [reply src/tenant-token/README.md:39]: Link to npm?

Comment (bidoubiwa) [reply src/tenant-token/README.md:39]: Yes!

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥🔥 Great Job

**Comment by shivaylamba** [comment]
Thank you @bidoubiwa

File: src/tenant-token/.gitignore [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#seed
+/seed/.env
+/seed/node_modules/
+/seed/data.ms/
+/seed/data.json
+
```
Comment (shivaylamba) [inline src/tenant-token/.gitignore:6]: Can you add your reviews here @mdubus

**Review by mdubus — COMMENTED** [review]
You did a really great job with this demo, it works very well and I absolutely love it 🤩
So big congrats to you 👏

Here are a few points that I noticed:
- There are multiple theming solutions used here (Tailwind, pure css, and styled-system). Should we go for just one ?
- There are a lot of useless code, which could be cleaned

I know that this is just a demo and we don't really want to spend too much time on it, but from my point of view, if I want to know how you did this demo and I check the code, it's really hard to understand what's going on because of the amount of code, which can be reduced (removal of useless code). 
Plus, this demo is also Meilisearch's image, and by checking the code I have the feeling that some code was copy-pasted from somewhere else (which is the case for some components). As I worked on the mini-dashboard I know that the code comes from internally, but it may not be obvious for some curious external users who may check the code of this demo.

Maybe @ferdi05 could help us take a decision about this points ?

I really hope that one day we will have a template for demos like this, it would be so much easier for everybody ! ☺️

**Comment by nicolasvienot** [comment]
Hello! Just passed by and I think there might be an issue with the Vercel configuration. You might want to check, this should display the folder `tenant-token` and not `src` :) Also could you rename the app from `demos-tenant-token` to `demo-tenant-token` in Vercel? Thanks!

Capture d’écran 2022-03-28 à 15 34 26

Great job @shivaylamba :fire:

**Pull request merged**